README.md

```diff
@@ -0,0 +1,56 @@
+# TAD Conversion Tool
+
+This repository contains transformations to the TAD System Card. Currently supported transformations:
+1. Algoritmeregister (v0.1.0, v0.4.0, v1.0.0) --> TAD System Card (v0.1a3).
+1. Toetsingskader algoritmes (v1) --> TAD Assessment Card (v0.1a3).
+
+## Prerequisites
+On MacOS install [yq](https://mikefarah.gitbook.io/yq) by executing the following commmand
+in the terminal:
+```
+brew install yq
+```
+
+To use `conversion_utils` Python 3.11 and Poetry are required.
+
+## Usage
+The directory `transformations/` contains the transformation files. These transformation files
+can be loaded by `yq` to transform a given input format to TAD System Cards. One can add transformations
+for new input formats.
+
+The directory `input/` contains the input files. Currently we support algoritmeregister (v0.1.0, v0.4.0, v1.0.0) and
+toetsingskader algoritmes (v1). These files can be manually obtained by using the `conversions_utils` Python script:
+```
```

Comment:
Could you add the scripttype here? (and other places in the README)

Reply:
What do you mean exactly with scripttype?

Reply:
i would like to have the quotes with a type:
'''shell for example